USAGE.md

```diff
@@ -31,16 +31,28 @@ To use the theme JSON file in your app, follow the steps below:
         - assets/appainter_theme.json
     ```
 
-4. Update your `main` function to load your theme. 
+4. Update your `main` function to be async, load your theme and pass it into your App
 
     ```dart
-    void main() async {
-      WidgetsFlutterBinding.ensureInitialized();
-      final themeStr = await rootBundle.loadString('assets/appainter_theme.json');
-      final themeJson = jsonDecode(themeStr);
-      final theme = ThemeDecoder.decodeThemeData(themeJson)!;
-    }
+      void main() async {
+        WidgetsFlutterBinding.ensureInitialized();
+
+        final themeStr = await rootBundle.loadString('assets/appainter_theme.json');
+        final themeJson = jsonDecode(themeStr);
+        final theme = ThemeDecoder.decodeThemeData(themeJson)!;
+
+        runApp(MyApp(theme: theme));
+      }
+
+      class MyApp extends StatelessWidget {
+        final ThemeData theme;
+
+        const MyApp({Key? key, required this.theme}) : super(key: key);
+        ....
     ```
+    Tips for those new to flutter/dart:
+    - `rootBundle` is available by importing `'package:flutter/services.dart';`
+    - `jsonDecode` is available by importing `'dart:convert';`
```

Comment:
Can you simply put the following imports as part of the code snippet? I think that might be clearer and easier for people to just copy and paste a working example.

```dart
import 'dart:convert';

import 'package:flutter/material.dart';
import 'package:flutter/services.dart';
import 'package:json_theme/json_theme.dart';
```